tests/tests.py

```
@@ -931,6 +931,24 @@ def test_groups_out_of_jobs():
     )
 
 
+def test_resource_is_ordinary_string():
+    # Ordinary strings
+    assert is_ordinary_string("hello") is True
```

Ruff (0.12.2)

Use of `assert` detected

(S101)